<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pdfEdit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="showArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>572</width>
           <height>499</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionAppend"/>
    <addaction name="actionExtract"/>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionMove"/>
    <addaction name="actionRotate"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>转换</string>
    </property>
    <addaction name="actionPicToPdf"/>
    <addaction name="actionMerge"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionAppend"/>
   <addaction name="actionExtract"/>
   <addaction name="actionCopy"/>
   <addaction name="actionMove"/>
   <addaction name="actionRotate"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionPicToPdf"/>
   <addaction name="actionMerge"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/preview.png</normaloff>:/res/res/preview.png</iconset>
   </property>
   <property name="text">
    <string>打开...</string>
   </property>
   <property name="toolTip">
    <string>打开pdf文档</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/save.png</normaloff>:/res/res/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前pdf文档</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/picturetopgallery.png</normaloff>:/res/res/picturetopgallery.png</iconset>
   </property>
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="toolTip">
    <string>pdf文档另存为其他文件名</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/print.png</normaloff>:/res/res/print.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="toolTip">
    <string>打印当前pdf文档</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/delete.png</normaloff>:/res/res/delete.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭pdf文档</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/exit.png</normaloff>:/res/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出系统</string>
   </property>
  </action>
  <action name="actionAppend">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/add.png</normaloff>:/res/res/add.png</iconset>
   </property>
   <property name="text">
    <string>追加</string>
   </property>
   <property name="toolTip">
    <string>追加一个pdf文档或图片到当前文档</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/del.png</normaloff>:/res/res/del.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除当前页面</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/forward.png</normaloff>:/res/res/forward.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>导出部分或全部页面为pdf文档或图片</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/sort.png</normaloff>:/res/res/sort.png</iconset>
   </property>
   <property name="text">
    <string>移动</string>
   </property>
   <property name="toolTip">
    <string>当前页面移动到其他位置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/info.png</normaloff>:/res/res/info.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/font.png</normaloff>:/res/res/font.png</iconset>
   </property>
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/copy.png</normaloff>:/res/res/copy.png</iconset>
   </property>
   <property name="text">
    <string>拷贝</string>
   </property>
   <property name="toolTip">
    <string>拷贝当前页面到剪贴板</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/convert.png</normaloff>:/res/res/convert.png</iconset>
   </property>
   <property name="text">
    <string>旋转</string>
   </property>
   <property name="toolTip">
    <string>当前旋转90度</string>
   </property>
  </action>
  <action name="actionPicToPdf">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/address.png</normaloff>:/res/res/address.png</iconset>
   </property>
   <property name="text">
    <string>图片转PDF</string>
   </property>
   <property name="toolTip">
    <string>图片转PDF工具</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="icon">
    <iconset resource="PdfRes.qrc">
     <normaloff>:/res/res/custom.png</normaloff>:/res/res/custom.png</iconset>
   </property>
   <property name="text">
    <string>合并PDF</string>
   </property>
   <property name="toolTip">
    <string>合并多个PDF</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="PdfRes.qrc"/>
 </resources>
 <connections/>
</ui>
